cnd/src/seed.rs

```diff
@@ -51,8 +191,14 @@ impl From<[u8; 32]> for Seed {
 #[cfg(test)]
 mod tests {
     use super::*;
+    use pem;
     use rand::rngs::OsRng;
 
+    #[test]
+    fn seed_byte_string_must_be_32_bytes_long() {
+        let _seed = Seed::from(*b"this string is exactly 32 bytes!");
```

Comment:
Are you testing the failure?

Reply:
Not like this but further down as part of pem string testing the failure case is tested.